src/joyosc/GameController.cpp

```diff
@@ -26,6 +26,28 @@
 #include "GameControllerIgnore.h"
 #include "Path.h"
 
+static const char *GetSensorName(SDL_SensorType sensor)
+{
+    switch (sensor) {
+    case SDL_SENSOR_ACCEL:
+        return "accelerometer";
+    case SDL_SENSOR_GYRO:
+        return "gyro";
+#if HAVE_DECL_SDL_SENSOR_ACCEL_L
+    case SDL_SENSOR_ACCEL_L:
+        return "accelerometer (L)";
```

Comment:
I would avoid names with spaces as it would be harder to match using Pd's [route] or [select] objects. If we follow how the button naming is done, ala "leftshoulder", we could have "accel", "gyro", "leftaccel", "leftgyro" etc. I am not a huge fan of the full "accelerometer" as I often misspell it, however the full name would match the full names using in the buttons ala "shoulder" and "trigger".

Reply:
You're right, will fix.